docs/source/getting-started/hello-opendp.rst

```diff
@@ -38,17 +38,13 @@ Enable ``contrib`` globally with the following snippet:
 
```

Comment:
The `install.packages` link above should go to `repos = 'https://opendp.r-universe.dev/'`

Reply:
resolved